TarponSimulator2017/Core/Boat.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.IO;
+using System.Collections;
+using System.Windows;
+
+using Microsoft.Xna.Framework.Input;
+
+namespace Core
+{
+	public class Boat
+	{
+		public Boat ()
+		{
+		}
+
+		public Vector Position
+		{
+			get { return _position; }
+			set { _position = value; }
+		}
+		protected Vector _position;
+
+		protected Vector Direction
+		{
+			get { return _direction; }
+			set { _direction = value; }
+		}
+		protected Vector _direction;
+
+		public int Speed
+		{
+			get { return _speed; }
+			set { _speed = value; }
+		}
+		protected int _speed;
+
+		public void Initialize()
+		{
+			_position = new Vector(0,0);
```

Comment:
guess it's the answer of my previous question, I do not really like that but if's the rule in C# I guess i have to respect it :)